### PR TITLE
added layout for pop person detail

### PR DESCRIPTION
**Pull request by alishechka**
-layout for popular person detail
-tools:text used with src image as sample

### CODE REVIEW

File: app/src/main/res/layout/activity_popular_people_detail.xml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/tv_popular_person_name"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/popular_person_detail_margin"
+        android:gravity="center"
```
Comment (EMHankombo) [inline app/src/main/res/layout/activity_popular_people_detail.xml:13]: Not needed with contraint layout

File: app/src/main/res/layout/activity_popular_people_detail.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+
+        tools:src="@drawable/popular_person_profile_image" />
+
+
+    <TextView
+        android:id="@+id/tv_popular_person_rating"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/popular_person_detail_margin"
+        android:gravity="center"
```
Comment (EMHankombo) [inline app/src/main/res/layout/activity_popular_people_detail.xml:43]: Neither is this

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -20,5 +20,6 @@
     <string name="txt_error_failed_registration">Registration failed :(</string>
     <string name="error_empty_field">Field cannot be empty!</string>
     <string name="txt_successful_registration">Registered successfully!</string>
+    <string name="actor_profile_photo">actor profile photo</string>
     >>>>>>> develop
```
Comment (EMHankombo) [inline app/src/main/res/values/strings.xml:24]: Remove this >>>>> develop

**Review by EMHankombo — CHANGES_REQUESTED** [review]
Address comments

File: app/src/main/res/layout/activity_popular_people_detail.xml (hunk 1 above)
Comment (alishechka) [reply app/src/main/res/layout/activity_popular_people_detail.xml:13]: android:gravity="center" removed

File: app/src/main/res/layout/activity_popular_people_detail.xml (hunk 2 above)
Comment (alishechka) [reply app/src/main/res/layout/activity_popular_people_detail.xml:43]: android:gravity="center" removed

File: app/src/main/res/values/strings.xml (hunk 3 above)
Comment (alishechka) [reply app/src/main/res/values/strings.xml:24]: removed

**Pull request closed without merging**